dev-docs/feature-conditional.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: page
+title: Conditional Ad Units - <font color="red">NEW!</font>
+description: Using labels for conditional bids
+---
+
+<div class="bs-docs-section" markdown="1">
+
+# Conditional Bids and AdUnits
+
+Prebid 1.0 introduced the `sizeConfig` feature primarily for [responsive ad designs]({{site.baseurl}}/dev-docs/feature-responsive.html), focusing on constraining sizes based on device.
```

Comment:
Now that 1.0 is out, I think we don't need to refer to it specifically, my preference would be something like 

<blockquote>
The <code>sizeConfig</code> property is used primarily for responsive ad designs ...  It supports a number of additional features, including: ...
</blockquote>